### DISCUSSION TITLE
Change font in every page

### DISCUSSION THREAD

**Discussion by quanghieu31** [luizdepra/hugo-coder#843] [Q&A]
Hi! I have been working on the hugo-coder theme and really love it. I only want to change the font to this Google Font ([link](https://fonts.google.com/specimen/Source+Sans+3#styles)). 

I have tried multiple ways to pull the Google Font API but all of them failed to have the new font appear on my website. My layouts folder looks very messy right now. Did you guys have a similar issue when changing the font? Any thought is appreciated. Thank you!

Here is my repo: [https://github.com/quanghieu31/quanghieu31.github.io](https://github.com/quanghieu31/quanghieu31.github.io)

**Comment by eeriemyxi** [comment]
> Hi! I have been working on the hugo-coder theme and really love it. I only want to change the font to this Google Font ([link](https://fonts.google.com/specimen/Source+Sans+3#styles)).
> 
> I have tried multiple ways to pull the Google Font API but all of them failed to have the new font appear on my website. My layouts folder looks very messy right now. Did you guys have a similar issue when changing the font? Any thought is appreciated. Thank you!
> 
> Here is my repo: https://github.com/quanghieu31/quanghieu31.github.io

Did you manage to find a solution to this?

**Comment by luizdepra** [comment]
Hey. I think you'll need to override `_variables.scss`, changing the related font-family variable. Also, add a `@import` statement to load the web font.
Because you'll be generating CSS from SCSS, you must use the extended binary of Hugo. The one with the Asset Pipeline capabilities.

**Reply by eeriemyxi** [reply]
Hi. That works, thank you.
![image](https://github.com/luizdepra/hugo-coder/assets/86738916/1f6b607b-af4d-480d-bfc4-557cb076b4f7)

Off-topic, but would you happen to know how to prevent the automatic capitalization of the page title 'BLOGS' to match exactly as it's configured? I personally prefer it as 'Blogs' rather than 'BLOGS'.

**Reply by luizdepra** [reply]
Yup. You'll need to change/remove [this CSS rule](https://github.com/luizdepra/hugo-coder/blob/main/assets/scss/_navigation.scss#L24).
